packages/msw-addon/src/initialize.d.ts

```diff
@@ -0,0 +1,14 @@
+import type { RequestHandler, SetupApi, LifeCycleEventsMap } from 'msw'
+import type { SetupWorker } from 'msw/browser'
+import type { SetupServer } from 'msw/node'
+
+export declare var api: SetupApi<LifeCycleEventsMap>
+
+export type InitializeOptions =
+  | Parameters<SetupServer['listen']>[0]
+  | Parameters<SetupWorker['start']>[0]
+
+export declare function initialize(
```

Comment:
This is really an ambiguous placeholder module used for local development, where we don't yet know which environment the code will compile to. This must never be shipped. This is why explicit environment TypeScript configs never even include this file.